Documentation/misc/.NET Core Shared Framework composition - A & D team ownership.md

```diff
@@ -0,0 +1,71 @@
+# Overview
+
+.NET Acquisition and deployment team will act as arbitrator and work in close liaison with Shared Framework owners. Team will own defining the guidelines to produce and consume a consistent shared framework across .NET Core product. This will greatly help in being proactive to the issues that may come up while composing the product and delivering it to the customers.
+ 
+# Ownership: 
+1. Identify, and report redundancy of  common assemblies carried in multiple Shared Frameworks.
+2. Drive discussions among Technology owners, team doesn't own the decision or its delivery.
```

Comment:
what decision or delivery are you talking about?  These terms seem too vague.

Reply:
Acquisition team owns reporting the assembly redundancy issue, and providing the guidance. Respective technology team ( Shared Framework owner ) owns making the required code change to resolve the issue.